extensions/amp-story/1.0/amp-story-system-layer.css

```diff
@@ -102,12 +104,17 @@
 
 .i-amphtml-story-page-progress-bar {
   background: rgba(255, 255, 255, 0.4) !important;
-  border-radius: 1px !important;
```

Comment:
What can we do to keep this? I know it looks weird because of the `scale` transform, but is there anything we can do to keep a border radius?

Reply:
Sure, I just thought we wanted squared segments. But also, didn't we have performance issues due to using `border-radius` on resizing elements? I guess it doesn't apply here since we're not animating the width, thus not causing a repaint? (Maybe that a special case for videos in `amp-story-page` 3-panels desktop?) Just making sure. /cc @newmuis

Reply:
Also, the segments look weird with the radius on full-bleed with the `border-radius`:
![image](https://user-images.githubusercontent.com/5449100/63527926-fdfcfa80-c4cf-11e9-9d3f-f5365a10fdeb.png)

Reply:
Without `border-radius`:
![image](https://user-images.githubusercontent.com/5449100/63528382-d0648100-c4d0-11e9-8269-61172cb1da63.png)

Reply:
Here's a screenshot of the original mock:
![image](https://user-images.githubusercontent.com/1492044/63528691-63052000-c4d1-11e9-89a5-42d0cad6e49c.png)

The `border-radius` issue we had on videos is unrelated here.
I know it looks weird because of the scaleX, but is there any way we can make it look good?

Reply:
Just confirming: the border radius issues we had were all just because the video itself was being clipped.  This should be fine.

Reply:
Resolution for this as seen with Hong: we keep the border radius when segmentsCount < MAX_SEGMENTS, and remove it when we trigger the overflow animation.